tests/unit/style/parsing/basic_shape.rs

```diff
@@ -77,6 +77,7 @@ fn test_border_radius() {
 
 #[test]
 fn test_circle() {
+    /*
```

Comment:
Open an issue and leave a comment linking to them 😄 

(or just link to a comment on this issue)